### PR TITLE
Support for AlmaLinux8

### PR DESCRIPTION
**Pull request by treenerd**
Support for Almalinux 8 on several roles and tests.

I was able to test some roles where packages are public available.
EDB related software I couldn't test, because I don't have access to those sources.

There is also a change in `setup_pgbouncer` role which is not 100% related to Almalinux8 support.
But to ensure that openssl is installed if `pgbouncer_ssl` is set to true.
Not 100% sure if this should be in this merge request. 
I just had issues in some situations with pgbouncer if openssl is not installed with the roles. 
I link the [diff](https://github.com/EnterpriseDB/edb-ansible/compare/master...treenerd:edb-ansible:master#diff-dd732ea85d966797b10b2db7ae677535a4ae0249d5c2ae44135bf942fea8a47e) here, so it is easier to identify. 

I didn't adapt the related Readme files, but I can do if this changes are welcome.

```
(.venv) ➜  tests git:(master) ✗ ./test-runner.py \
  --edb-enable-repo false \
  --pg-version 14 \
  --os almalinux8 \
  --ansible-core-version 2.14 \
  --pg-type PG \
  --jobs 4 \
  -k setup_repo \
  install_dbserver \
  init_dbserver \
  manage_dbserver \
  setup_replication \
  setup_pgpool2 \
  manage_pgpool2 \
  setup_pgbouncer \
  manage_pgbouncer \
  setup_barmanserver \
  setup_barman \
  setup_hammerdb \
  setup_dbt2_driver \
  setup_dbt2_client \
  setup_dbt2 \
  setup_dbt3 \
  setup_dbt7 \
  setup_pgbackrestserver \
  setup_pgbackrest
Test setup_repo with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test init_dbserver with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test install_dbserver with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test manage_pgpool2 with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_replication with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test manage_dbserver with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test manage_pgbouncer with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_pgbouncer with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_pgpool2 with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_barmanserver with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_barman with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_hammerdb with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_dbt2_driver with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_dbt2_client with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_pgbackrestserver with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_dbt7 with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_dbt2 with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_pgbackrest with ansible-core v2.14 PG/14 on almalinux8 ... OK
Test setup_dbt3 with ansible-core v2.14 PG/14 on almalinux8 ... OK

Tests passed: 19/19 100.00%
```

### CODE REVIEW

**Comment by treenerd** [comment]
Ups..

I think I think I accidently merged this file in the wrong way:
[roles/setup_hammerdbserver/defaults/main.yml](https://github.com/EnterpriseDB/edb-ansible/compare/master...treenerd:edb-ansible:master#diff-bbf623ce19cb954b67af101fee627095ecec6d6749229f796cff121f62299ca2)

Sorry, I saw it a bit to late.

File: roles/setup_hammerdbserver/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+hammerdb_version: 4.6
+hammerdb_filename: "HammerDB-{{ hammerdb_version }}-\
+                    {{ 'RHEL' if ansible_os_family == 'RedHat' else 'Linux' }}\
+                    {{ ansible_distribution_major_version if \
+                    ansible_os_family == 'RedHat'}}.tar.gz"
+hammerdb_url: "https://github.com/TPC-Council/HammerDB/releases/download/"
+supported_os:
+  - CentOS8
+  - RHEL8
+  - Rocky8
```
Comment (jt-edb) [inline roles/setup_hammerdbserver/defaults/main.yml:11]: Please undo these changes on this file. I think you just have to rebase your branch.

**Review by jt-edb — COMMENTED** [review]
Thank you for this PR, please see my comment.

File: roles/setup_hammerdbserver/defaults/main.yml (hunk 1 above)
Comment (treenerd) [reply roles/setup_hammerdbserver/defaults/main.yml]: I removed the file and rebased it.
Sorry for the confusion.

Comment (jt-edb) [reply roles/setup_hammerdbserver/defaults/main.yml]: Looks good. Could you please update the main README file to reflect that almalinux8 is now supported? Thank you.

Comment (treenerd) [reply roles/setup_hammerdbserver/defaults/main.yml]: I added the information into the main README file as suggested.
